docs/content/languages/python.md

```diff
@@ -0,0 +1,84 @@
+---
+title: "Using Python with Fission"
```

Comment:
Since we probably want each language to have the same title structure what about moving the language to the end of the title. e.g. 
```
Fission Functions in Python
Fission Functions in Java
Fission Functions in Golang
```

Reply:
Or try to add serverless in here for SEO points.

Reply:
I see no benefit to removing the language name from the headings